mappings/markdown/Zeek/conn_log/README.md

```diff
@@ -1,55 +1,73 @@
 # Event Dossier: Zeek conn.log
-### 
+### Summary:
 - **Description**: Translates a Zeek conn.log to OCSF. 
 
 - **Event References**:
+  - https://schema.ocsf.io/1.3.0/classes/network_activity
   - https://docs.zeek.org/en/master/logs/conn.html
-  - https://schema.ocsf.io/1.0.0-rc.3/classes/network_activity
+  - https://docs.zeek.org/en/master/scripts/base/protocols/conn/main.zeek.html#type-Conn::Info
  
- ### OCSF Version: 1.0.0-rc.3
- - `category_uid`: `4`
+ ### Static mapping: OCSF Version 1.3.0
+ - `metadata.version`: `1.3.0`
  - `category_name`: `Network Activity`
- - `class_uid`: `4001`
+ - `category_uid`: `4`
  - `class_name`: `Network Activity`
- - `metadata.profiles`: `[security_control]`
- - `metadata.product.name`: `Zeek`
+ - `class_uid`: `4001`
+ - `metadata.log_name`: `conn.log`
+ - `metadata.loggers.log_provider`: `Zeek`
+ - `metadata.loggers.product.name=`: `Zeek`
```

Comment:
Two things come to mind here:

1. `metadata.loggers` is an array, so indexing it with `.` suggests a record. You could do `metadata.loggers[].log_provider`. Curious if we have a canonical way to index in text.
2. The `meta_data.log_*` fields feel more apt than using the `loggers` array. Any reason why not use `metadata.log_provider`?